src/emqx_mod_topic_metrics.erl

```diff
@@ -41,19 +41,19 @@
         ]).
 
 -export([ inc/2
-        , inc/3
-        , val/2
-        , metrics/1
-        , register/1
-        , unregister/1
-        , unregister_all/0
-        , is_registered/1
-        , all_registered_topics/0
-        ]).
+    , inc/3
+    , val/2
+    , metrics/1
+    , register/1
+    , unregister/1
+    , unregister_all/0
+    , is_registered/1
+    , all_registered_topics/0
+]).
 
 %% Exposed for test only.
```

Comment:
This was my bad.
`rate/2` is not exposed only for tests. I'll move it back to the old place.